RELEASE_NOTES.md

```diff
@@ -1,5 +1,152 @@
+- The Libpod Untag endpoint for images will now remove all tags for the given image if no repository and tag are specified for removal.
+- Fixed a bug where the Compat Logs endpoint for containers did not send a newline character after each log line.
+- Fixed a bug where the Compat Logs endpoint for containers would mangle line endings to change newline characters to add a preceding carriage return ([#7942](https://github.com/containers/podman/issues/7942)).
+- Fixed a bug where the Compat Inspect endpoint for Containers did not properly list the container's stop signal ([#7917](https://github.com/containers/podman/issues/7917)).
+- Fixed a bug where the Compat Inspect endpoint for Containers formatted the container's create time incorrectly ([#7860](https://github.com/containers/podman/issues/7860)).
+- Fixed a bug where the Compat Inspect endpoint for Containers did not include complete network information on the container.
+- Fixed a bug where the server could panic if a client closed a connection midway through an image pull ([#7896](https://github.com/containers/podman/issues/7896)).
+- Fixed a bug where the Compat Create endpoint for volumes returned an error when a volume with the same name already existed, instead of a 201 ([#7740](https://github.com/containers/podman/issues/7740)).
+- Fixed a bug where a client disconnecting from the Libpod or Compat events endpoints could result in the server using 100% CPU ([#7946](https://github.com/containers/podman/issues/7946)).
+- Fixed a bug where the "no such image" error message sent by the Compat Inspect endpoint for Images returned a 404 was improperly formatted for Docker compatibility.
+- Fixed a bug where the Compat Create endpoint for networks did not properly set a default for the `driver` parameter if it was not provided by the client.
+- Fixed a bug where the Compat Inspect endpoint for images did not populate the `RootFS` field of the response.
+
+### Misc
+- Updated Buildah to v1.18.0
+- Updated the containers/storage library to v1.24.0
+- Updated the containers/image library to v5.8.0
+- Updated the containers/common library to v0.27.0
+
+## 2.1.1
+### Changes
+- The `podman info` command now includes the cgroup manager Podman is using.
+
+### Bugfixes
+- Fixed a bug where Podman would not build with the `varlink` build tag enabled.
+- Fixed a bug where the `podman save` command could, when asked to save multiple images, write its progress bar to the archive instead of the terminal, producing a corrupted archive.
+- Fixed a bug where the `json-file` log driver did not write logs.
+- Fixed a bug where `podman-remote start --attach` did not properly handle detaching using the detach keys.
+- Fixed a bug where `podman pod ps --filter label=...` did not work.
+- Fixed a bug where the `podman build` command did not respect the `--runtime` flag.
```

Comment:
```suggestion
- Fixed a bug where the `podman build` command did not respect the `--runtime` option.
```

Reply:
These are the 2.1.1 release notes and are already released - they're basically set in stone at this point, so we can't fix them.